frontend/src/components/home/index.tsx

```diff
@@ -1,11 +1,23 @@
 import React from 'react';
+import { Route, Redirect, BrowserRouter as Router } from 'react-router-dom';
 import Sidebar from '../sidebar';
+import Employees from '../employees';
+import Rota from '../rota';
+import Summary from '../summary';
 import homeStyles from './home.module.scss';
 
 const Home = () => (
-  <div className={homeStyles.sidebar}>
-    <Sidebar />
-  </div>
+  <Router>
+    <div className={homeStyles.home}>
+      <Route path="/employees" component={Employees} />
+      <Route path="/rota" component={Rota} />
+      <Route path="/summary" component={Summary} />
+      <Route path="/" component={Sidebar} />
+    </div>
```

Comment:
Is there a reason we have the sidebar inside the router? I know nothing about best practices but what if we had the sidebar outside the router (given its always there so the two dont need to talk)

Reply:
Sidebar contains <Link> elements that redirect to other pages and must be inside router.
Otherwise I think I'd have to do something ugly like write some javascipt to change the url manually.

Reply:
Could the router in the `App` component instead of `Home`? That way all components in Home would be inside Router, or would this not work?